lapack/testlapack/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// testlapack implements a set of testing routines for Lapack functions.
```

Comment:
`// Package testlapack...`

This convention is important; the automation depends on at least some hook for safety.